src/components/anchor-navigation/anchor-navigation.spec.tsx

```diff
@@ -144,26 +147,29 @@ describe("AnchorNavigation", () => {
       });
       wrapper.update();
       expectNavigationItemToBeSelected(index, wrapper);
-
-      if (hasFocusableElement) {
```

Comment:
Split this test into two, one for testing the click and scroll to section, and the other click and focus on sections with focusable content, since eslint does not recommend conditional expect statements.